api-tests/plugins/i18n/content-manager/crud.test.api.js

```diff
@@ -111,20 +112,27 @@ describe('i18n - Content API', () => {
           method: 'POST',
           url: `/content-manager/collection-types/api::category.category`,
           qs: {
-            plugins: { i18n: { locale, relatedEntityId: data.categories[0].id } },
```

Comment:
do you know in what way we were using the `relatedEntityId` . I actually forgot why we had that 🤔

Reply:
see my comment here [`1c6897c` (#19153)](https://github.com/strapi/strapi/pull/19153/commits/1c6897cdf8a8b1ce4ff479da57d3265db14c3004)

wdyt?

Reply:
makes sesnse :)